sonar-ws/src/main/gen-java/org/sonarqube/ws/QualityProfiles.java

```diff
@@ -0,0 +1,2483 @@
+          bitField0_ |= 0x00000010;
+          isInherited_ = value;
+          onChanged();
+          return this;
+        }
+        /**
+         * <code>optional bool isInherited = 5;</code>
+         */
+        public Builder clearIsInherited() {
+          bitField0_ = (bitField0_ & ~0x00000010);
+          isInherited_ = false;
+          onChanged();
+          return this;
+        }
+
+        private java.lang.Object parentKey_ = "";
+        /**
+         * <code>optional string parentKey = 6;</code>
+         */
+        public boolean hasParentKey() {
+          return ((bitField0_ & 0x00000020) == 0x00000020);
+        }
+        /**
+         * <code>optional string parentKey = 6;</code>
+         */
+        public java.lang.String getParentKey() {
+          java.lang.Object ref = parentKey_;
+          if (!(ref instanceof java.lang.String)) {
+            com.google.protobuf.ByteString bs =
+                (com.google.protobuf.ByteString) ref;
+            java.lang.String s = bs.toStringUtf8();
+            if (bs.isValidUtf8()) {
+              parentKey_ = s;
+            }
+            return s;
+          } else {
+            return (java.lang.String) ref;
+          }
+        }
+        /**
+         * <code>optional string parentKey = 6;</code>
+         */
+        public com.google.protobuf.ByteString
+            getParentKeyBytes() {
+          java.lang.Object ref = parentKey_;
+          if (ref instanceof String) {
+            com.google.protobuf.ByteString b = 
+                com.google.protobuf.ByteString.copyFromUtf8(
+                    (java.lang.String) ref);
+            parentKey_ = b;
+            return b;
+          } else {
+            return (com.google.protobuf.ByteString) ref;
+          }
+        }
+        /**
+         * <code>optional string parentKey = 6;</code>
+         */
+        public Builder setParentKey(
+            java.lang.String value) {
+          if (value == null) {
+    throw new NullPointerException();
+  }
+  bitField0_ |= 0x00000020;
+          parentKey_ = value;
+          onChanged();
+          return this;
+        }
+        /**
+         * <code>optional string parentKey = 6;</code>
+         */
+        public Builder clearParentKey() {
+          bitField0_ = (bitField0_ & ~0x00000020);
+          parentKey_ = getDefaultInstance().getParentKey();
+          onChanged();
+          return this;
+        }
+        /**
+         * <code>optional string parentKey = 6;</code>
+         */
+        public Builder setParentKeyBytes(
+            com.google.protobuf.ByteString value) {
+          if (value == null) {
+    throw new NullPointerException();
+  }
+  bitField0_ |= 0x00000020;
+          parentKey_ = value;
+          onChanged();
+          return this;
+        }
+
+        private java.lang.Object parentName_ = "";
+        /**
+         * <code>optional string parentName = 7;</code>
+         */
+        public boolean hasParentName() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add the "@Override" annotation above this method signature [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1161)